cmd/nscgc/main.go

```diff
@@ -92,15 +93,29 @@ func run(ctx context.Context) error {
 		return fmt.Errorf("failed to make registry: %w", err)
 	}
 
-	olderThan := time.Now().Add(-3 * 24 * time.Hour)
-
-	if err := storage.MarkAndSweep(ctx, driver, registry, storage.GCOpts{
-		DryRun:           *dryRun,
-		RemoveUntagged:   *removeUntagged,
-		OlderThan:        olderThan,
-		ExhaustiveNeeded: exhaustiveNeeded,
-	}); err != nil {
-		return fmt.Errorf("failed to garbage collect: %v", err)
+	if useManifestBasedGC != nil && *useManifestBasedGC {
+		// cleanup after 24h
+		olderThan := time.Now().UTC().Add(-24 * time.Hour)
+
+		if err := storage.MarkAndSweepManifests(ctx, driver, registry, storage.GCOpts{
+			DryRun:           *dryRun,
+			RemoveUntagged:   *removeUntagged,
+			OlderThan:        olderThan,
+			ExhaustiveNeeded: exhaustiveNeeded,
+		}); err != nil {
+			return fmt.Errorf("failed to garbage collect: %v", err)
+		}
+	} else {
+		olderThan := time.Now().Add(-3 * 24 * time.Hour)
```

Comment:
Not sure if the UTC part is important but it made debugging much easier

Reply:
can you add UTC to the other branch too please? (for consistency)